wordpress-mobile/WordPress-Android
# Adds group field in the starter designs response

## Description
This PR updates the FluxC reference to add the new vertical field in the starter designs response. [This intents to be used to filter and recommend designs based on the vertical selection of the user during site creation](https://github.com/wordpress-mobile/WordPress-Android/issues/16503).

Note:
This PR was created [to be able to merge the FluxC change](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2385) without waiting for[ the full Site Design Revamp feature branch to be merged](https://github.com/wordpress-mobile/WordPress-Android/pull/16468).

## To test:
1. Verify that the Site Design Picker works without issues
2. Verify that the new field is available on debug mode (for more extensive functional test check the Site Design Revamp Android PR: https://github.com/wordpress-mobile/WordPress-Android/pull/16510)

## Regression Notes
1. Potential unintended areas of impact
N/A

3. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

4. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
